.gitignore

```diff
@@ -26,3 +26,54 @@ node_modules
 
 # Users Environment Variables
 .lock-wscript
+
+.gitignore
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion
+
+*.iml
+
+## Directory-based project format:
+.idea/
+# if you remove the above rule, at least ignore the following:
```

Comment:
I used a plugin for IntelliJ to generate these settings I actually never looked at them :-)
I would say leave the *.iml and the .idea/ and remove the rest...